cocos/2d/utils/text-utils.ts

```diff
@@ -78,14 +78,14 @@ export class LRUCache {
     private count = 0;
     private limit = 0;
     private datas: Record<string, ICacheNode> = {};
-    private declare head;
-    private declare tail;
+    private head: ICacheNode | null = null;
+    private tail: ICacheNode | null = null;
```

Comment:
Why not use declare?

Reply:
Because `head` and `tail` is not assigned in `constructor`, so we need to initialize them to `null` while definition.